parsers/src/main/scala/uk/gov/ons/addressIndex/crfscala/CrfScalaJni.scala

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ons.addressIndex.crfscala
+
+//todo scaladoc
+trait CrfScalaJni {
+  /**
+    *
+    * @param modelPath
+    * @param items
+    * @return
+    */
+  def tag(modelPath: String, items : String) : String
+}
+
+class CrfScalaJniImpl extends CrfScalaJni {
+  /**
+    *
+    * @param modelPath
+    * @param items
+    * @return
+    */
+  @native def tag(modelPath: String, items : String) : String
+}
+
+object CrfScalaJni {
+  val tab = "\t"
```

Comment:
@Mironor This is not a Scala default, constants are merely capitalised, but not entirely uppercased in scala. `Constant` not `CONSTANT`, by convention.